### ISSUE TITLE
Lollipop: error: only position independent executables (PIE) are supported.

### ISSUE THREAD

**Issue by m-o** [cine-io/android-ffmpeg-with-rtmp#14]
Hi, I have build the ffmpeg library and copied all files in /dist to /files folder in my android app.
Previously, I have been able to run ffmpeg without ramp support via adb shell.
I am running Android 5.0.1.
Bud when I try to run this build, I get error:
shell@falcon_umts:/data/data/xyz/files/ff/bin $ ./ffmpeg
error: only position independent executables (PIE) are supported.
(I have set all the appropriate permissions)

I thing it is connected to this Issue: https://code.google.com/p/android-developer-preview/issues/detail?id=888 mentioned here: https://isocpp.org/blog/2014/12/android-ndk-revision-10d-available.
But my skills are not sufficient to change the build.sh script with flags and understand what I am doing.

Have you encountered this issue? Could you update .sh file to fix this?
Thank you!

**Comment by lgorse** [comment]
Hey @m-o  did you ever solve this? I run into the same issue with the compiled files and am having trouble getting to the bottom.

**Comment by m-o** [comment]
No, we decided to buy library with working FFMpeg and streaming using NDK.
